src/Route/DataAccess.ts

```diff
@@ -20,10 +21,12 @@ export async function getRoutes(requests: RouteRequest[]): Promise<Route[]> {
 			requests[index].chosenMode,
 			requests[index].type
 		);
-		route.modeDirections.push({
-			mode: 'plane',
-			directions: [estimatePlaneDirection(route.origin, route.destination)]
-		});
+		if (getDistance(route.origin, route.destination, EARTH_RADIUS) > 100000) {
```

Comment:
We already have a function for that. `getAvailableModes` in ModeSelector.ts so please use that one.
If you want to put i into Dao, remove it from Route/index.ts. The original idea was that DAO returns everything it can and some filtering will be done on some upper layer. But in the case of airplane I think it is ok, that dao returns it when it males sence.